packages/core/src/render3/node_manipulation.ts

```diff
@@ -863,3 +731,158 @@ function appendProjectedNode(
     }
   }
 }
+
+
+/**
+ * `executeActionOnView` performs operation on the view as specified in `action` (insert, detach,
```

Comment:
Same for the comments below

```suggestion
 * `executeActionOnView` performs an operation on the view as specified in `action` (insert, detach,
```